src/test/java/org/broadinstitute/hellbender/utils/clipping/ReadClipperUnitTest.java

```diff
@@ -387,4 +392,60 @@ public void testRevertEntirelySoftclippedReads() {
         Assert.assertEquals(clippedRead.getStart(), getSoftStart(read));
     }
 
+
+    @Test
+    public void testSoftClipBothEndsByReferenceCoordinates() {
+        for (Cigar cigar : cigarList) {
+            GATKRead read = ReadClipperTestUtils.makeReadFromCigar(cigar);
+            int alnStart = read.getStart();
+            int alnEnd = read.getEnd();
+            int readLength = alnStart - alnEnd;
+            for (int i = 0; i < readLength / 2; i++) {
+                GATKRead clippedRead = ReadClipper.softClipBothEndsByReferenceCoordinates(read, alnStart + i, alnEnd - i);
+                Assert.assertTrue(clippedRead.getStart() >= alnStart + i, String.format("Clipped alignment start is less than original read (minus %d): %s -> %s", i, read.getCigar().toString(), clippedRead.getCigar().toString()));
+                Assert.assertTrue(clippedRead.getEnd() <= alnEnd + i, String.format("Clipped alignment end is greater than original read (minus %d): %s -> %s", i, read.getCigar().toString(), clippedRead.getCigar().toString()));
```

Comment:
Are the comments on these two asserts the opposite of what they should be? Clipped alignment start is greater than original and clipped alignment ed is less than the original.